integration/e2e/e2e_test.go

```diff
@@ -126,6 +126,8 @@ func TestAllInOne(t *testing.T) {
 			now := time.Now()
 			util.SearchAndAssertTraceBackend(t, apiClient, info, now.Add(-20*time.Minute).Unix(), now.Unix())
 
+			util.SearchAndAsserTagsBackend(t, apiClient, now.Add(-20*time.Minute).Unix(), now.Unix())
```

Comment:
It'd be awesome if we could integrate tests for this change in `api_test.go`. There are tests in that file that cover the tags and tag values APIs.